files/es/web/javascript/reference/operators/addition_assignment/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Asignación de adición (+=)
+slug: Web/JavaScript/Reference/Operators/Addition_assignment
+---
+
+{{jsSidebar("Operators")}}
+
+El **operador de asignación de adición (`+=`)** realiza una [suma](/es/docs/Web/JavaScript/Reference/Operators/Addition) (que puede ser una suma numérica o una concatenación de cadenas) en los dos operandos y asigna el resultado al operando izquierdo.
+
+{{EmbedInteractiveExample("pages/js/expressions-addition-assignment.html")}}
+
+## Sintaxis
+
+```js-nolint
+x += y
+```
+
+## Descripción
+
+`x += y` es equivalente a `x = x + y`, exepto que la expresión `x` se evalúa solo una vez.
+
+## Ejemplos
+
+### Usando el operador de asignación de adición
+
+```js
+let baz = true;
+
+// Booleano + Número -> adición
+baz += 1; // 2
+
+// Número + Booleano -> adición
+baz += false; // 2
+```
+
+```js
+let foo = "foo";
+
+// Cadena de texto + Booleano -> concatenación
+foo += false; // "foofalse"
+
+// Cadena de texto + Cadena de texto -> concatenación
+foo += "bar"; // "foofalsebar"
+```
+
+```js
+let bar = 5;
+
+// Número + Número -> adición
+bar += 2; // 7
+
+// Número + Cadena de texto -> concatenación
+bar += "foo"; // "7foo"
+```
+
+```js
+let x = 1n;
+
+// BigInt + BigInt -> adición
+x += 2n; // 3n
+
+// BigInt + Número -> devuelve TypeError
+x += 1; // TypeError: Cannot mix BigInt and other types, use explicit conversions
+```
+
+## Especificaciones
+
+{{Specifications}}
+
+## Compatibilidad con navegadores
+
+{{Compat}}
+
+## Véase también
+
+- [Operadores de asignación en la guía JS](/es/docs/Web/JavaScript/Guide/Expressions_and_operators#assignment_operators)
+- [Adición (`+`)](/es/docs/Web/JavaScript/Reference/Operators/Addition)
```

Comment:
Hi!
I don't understand the error, GitHub doesn't let me commit the suggestion because "it does not have any change", can somebody please help me fix it?
![image](https://github.com/mdn/translated-content/assets/83504427/048dd2cd-fab1-4f7c-87ea-4ade7c125ea3)

Reply:
The final character of the document needs to be a `newline` character, it is a invisible character thats why we cannot see it

This is the error message: 
```
Files should end with a single newline character
```

Reply:
Thanks! And how can I do this? Sorry, I'm learning and I want to know for future contributions to this repo.

Reply:
GitHub is a little weird with this, but in your last PR, a just edited the bot observation and put a blank newline, line 78 and commit the changes

Reply:
Thanks! I'll remember it for future contributions